pkg/modules/libreoffice/routes.go

```diff
@@ -157,6 +179,18 @@ func convertRoute(libreOffice libreofficeapi.Uno, engine gotenberg.PdfEngine) ap
 				outputPaths = convertOutputPaths
 			}
 
+			// Writes metadata specified in the user request to the output PDF.
+
+			// Note: Any matching existing metadata entries (identified by key)
+			// will be overwritten by user specified entries during this operation.
+
+			if len(metadata) > 0 {
+				err = engine.WriteMetadata(ctx, ctx.Log(), outputPaths, metadata)
+				if err != nil {
+					return fmt.Errorf("write metadata failure: %w", err)
+				}
```

Comment:
```suggestion
				err = engine.WriteMetadata(ctx, ctx.Log(), outputPaths, metadata)
				if err != nil {
					return fmt.Errorf("write metadata: %w", err)
				}
```

Some loop required there 😄

Reply:
The single error returned has a list of all files which got error while writing metadata.